src/main/java/no/fdk/dataservicecatalog/config/WebConfigProperties.java

```diff
@@ -0,0 +1,12 @@
+package no.fdk.dataservicecatalog.config;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Data
+@Component
+@ConfigurationProperties("application")
+public class WebConfigProperties {
+    private String frontendUrl;
+}
```

Comment:
Questionnable naming. Rename frontendUrl to dataServiceCatalogGuiUrl or something similar. Also, move it to an already existing ApplicationProperties class